src/ero_mmnet.py

```diff
@@ -106,6 +113,39 @@ def import_circles(self, ego_node_id, ego_network_circles_path):
 
             self.circles[ego_node_id] = circles
 
+    def import_events(self, events_filename):
+        parser = event_parser.EventParser()
+        events_json = json.load(open(events_filename))['data']
+        events_mode = self.mmnet.GetModeNetByName("Event")
+        for event in events_json:
+            parsed_event = parser.parse_event(event)
+            self.events[parsed_event.id] = event
+            events_mode.AddNode(parsed_event.id)
+            self.connect_event_in_network(parsed_event)
+
+    def connect_event_in_network(self, event):
+        neighbors = self.generate_random_neighbors()
+        crossnet_person_to_event = self.mmnet.GetCrossNetByName(
+            "EventToPerson")
+        for neighbor in neighbors:
+            edge_id = crossnet_person_to_event.AddEdge(event.id, neighbor)
+            self.event_to_person_edges[edge_id] = (
+                event.id, neighbor)
+
+    def generate_random_neighbors(self):
```

Comment:
Since this generation is meant to be a workaround to the missing links in the data, it would logically be an external tool to the class. A class EventNeighboursGenerator with a method that accepts the reference to the list of people and events and returns the list of neighbours ids can do the work. Btw, this could be left as an open issue.